packages/theming/theming-utils/src/mapPipelineToTheme.android.ts

```diff
@@ -24,6 +24,9 @@ export function mapPipelineToTheme(pipelineOutput: any): AliasColorTokens {
     neutralForeground3BrandPressed: pipelineOutput.neutralForeground3.fillColorBrandPressed,
     neutralForeground3BrandSelected: pipelineOutput.neutralForeground3.fillColorBrandSelected,
     neutralForegroundInverted: pipelineOutput.neutralForegroundInverted.fillColorRest,
+    neutralForegroundDisabled: pipelineOutput.neutralForegroundDisabled1.fillColorRest,
+    neutralForegroundDisabled1: pipelineOutput.neutralForegroundDisabled2.fillColorRest,
```

Comment:
I feel these tokens should be either called ('neutralForegroundDisabled' and 'neutralForegroundDisabled2') or ('neutralForegroundDisabled1' and 'neutralForegroundDisabled2'), current naming is misleading for 'neutralForegroundDisabled1'

Reply:
Valid, updated.